src/components/create-safe/status/hooks/useWatchSafeCreation.ts

```diff
@@ -75,7 +75,11 @@ const useWatchSafeCreation = ({
       }
     }
 
-    if (status === SafeCreationStatus.ERROR || status === SafeCreationStatus.REVERTED) {
+    if (
+      status === SafeCreationStatus.ERROR ||
+      status === SafeCreationStatus.REVERTED ||
+      status === SafeCreationStatus.TIMEOUT
```

Comment:
Since we also display the Retry button on `TIMEOUT` it makes sense to clear the `txHash` then as well.